content/statefulset/testfailure.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Test Failure"
+date: 2018-08-07T08:30:11-07:00
+weight: 30
+---
+#### Unhealthy container
+MySQL container uses readiness probe by running mysql -h 127.0.0.1 -e 'SELECT 1' on the server to make sure MySQL server is still active.
+Open a new terminal and simulate MySQL as being unresponsive by following command.
+```
+kubectl exec mysql-2 -c mysql -- mv /usr/bin/mysql /usr/bin/mysql.off
+```
+This command renames the /usr/bin/mysql command so shat readiness probe can't find it. During the next health check, the pod should report one of it's containers is not healthy. This can be verified by following command.
```

Comment:
s/shat/that/ maybe?

Reply:
I just changed so shat -> so that
Thank you!!